Gems/AWSCore/Code/Include/Framework/HttpRequestJob.h

```diff
@@ -183,8 +183,7 @@ namespace AWSCore
         {
             AZ_UNUSED(response);
         };
-
-    private:
```

Comment:
Intentionally `private` to keep `Process()` protected, so that child classes can build perf or other functionality around it.